repos/spack_repo/builtin/packages/py_dask_mpi/package.py

```diff
@@ -17,6 +17,16 @@ class PyDaskMpi(PythonPackage):
 
     license("BSD-3-Clause")
 
+    # This is not an official release
+    # There have been no releases post-2022, c.f.,
+    # https://github.com/dask/dask-mpi/issues/136
+    # but various fixes have been implimented so this is
+    # the most recent main commit as of 2025-09-26
+    version(
+        "2025.0.0",
+        sha256="39ee0564af77be6eb34e52581dafbb1335ba82d0fccd2d7fa6182e530251a19d",
+        url="https://www.github.com/dask/dask-mpi/tarball/3d4742e7fca906e265234fb48360990eafe51950",
+    )
```

Comment:
I would instead add:
```python
git = "https://github.com/dask/dask-mpi.git"
```
to the top and change this to:
```suggestion
    version("2025.4.1", commit="3d4742e7fca906e265234fb48360990eafe51950")
```
This is the date and hash of the most recent commit.

I would also be fine with adding:
```python
version("main", branch="main")
```
as well if that makes things easier.

Reply:
Oh ok, I like this more thanks